client/chat/action-creators.js

```diff
@@ -80,7 +84,10 @@ export function getMessageHistory(channel, limit) {
     })
     dispatch({
       type: CHAT_LOAD_CHANNEL_HISTORY,
-      payload: siteSocket.invoke('/chat/getHistory', params),
+      payload: fetch(
+        apiUrl`chat/${channel}/chat?limit=${limit}&beforeTime=${earliestMessageTime}`,
```

Comment:
And yeah, think this URL should match the one we use for posting a message, so `/chat/<channel>/messages` or `/chat/<channel>/chat`, one of the two (I vaguely prefer `messages` because it is less redundant/filled with multiple meanings for the same word :) )

Feel free to put these changes in the previous commit if you prefer